pkg/service/system/route53.go

```diff
@@ -129,3 +182,176 @@ func AddRoute53Record(ctx context.Context, dnsName, recordName, regionName strin
 
 	return *result.ChangeInfo.Id, nil
 }
+
+// GetRoute53TXTRecords will return TXT records only
+func GetRoute53TXTRecords(ctx context.Context) ([]types.Route53Record, error) {
+
+	hostedZoneId := config.Get().AwsRoute53HostedZoneID
+
+	// Creating AWS Route53 session
+	route53Client, err := getRoute53Sess(defaultRegion)
+	if err != nil {
+		return nil, errors.Wrap(err, "GetRoute53Records: failed to create route53 session")
+	}
+
+	getRecordsInput := &route53.ListResourceRecordSetsInput{
+		HostedZoneId: aws.String(hostedZoneId),
+		MaxItems:     aws.String("1000"),
+	}
+
+	var records []types.Route53Record
+	var recordSets []*route53.ResourceRecordSet
+
+	for {
```

Comment:
do comment this section, the paging is not quite transparent.

Reply:
ok, will add in next commit